### ISSUE TITLE
Hide "Subs" button when no subtitles are available

### ISSUE THREAD

**Issue by christf** [sosedoff/omxremote#19]
When no subtitles are available, the button could be hidden to avoid confusing the user into thinking it is not working.
Similarly the audio track selection buttons could be hidden if there is only one audio track

**Comment by sosedoff** [comment]
Yeah, the frontend is pretty dumb right now, and always been like that. However, i never had an issue with it because i always know what in my video queue. Feel free to add the check of some sorts.

Side note: I actually stopped using the web view years ago and built a native iOS app that talks to the API instead. Not really interested in adding more UI stuff in the web view, but im happy to test and review PRs when i have time.

**Comment by christf** [comment]
unfortunately iOS apps do not work on android. Is the frontend written in kotlin by chance?

**Comment by sosedoff** [comment]
Its built in swift, and also hasn't seen a change in years :)
Not that i have abandoned the project, i just rarely use it.
